css_tasks/task10/js/app.js

```diff
@@ -0,0 +1,6 @@
+import {submitForm} from './form.js';
+import {DOMElements} from './elements.js';
+const {signupForm} = DOMElements;
+
+// EVENTS
+signupForm.addEventListener('submit', submitForm);
```

Comment:
any access to your DOM should be wrapped to DOMContentLoaded callback.
Not each one, but the whole script that's going to access DOM